Userland/Libraries/LibVideo/VP9/Parser.cpp

```diff
@@ -1736,6 +1729,7 @@ void Parser::dump_info()
     outln("Frame dimensions: {}x{}", m_frame_width, m_frame_height);
     outln("Render dimensions: {}x{}", m_render_width, m_render_height);
     outln("Bit depth: {}", m_bit_depth);
+    outln("Show frame: {}", m_show_frame);
```

Comment:
I feel like this was removed super early in the commit stack and just added back now in `LibVideo: Remove unnecessary dbgln calls` ? if so, the addition should time travel back to the commit that removed it.

Reply:
Actually, the removed line previously was printing the interpolation filter, which was uninitialized memory for the keyframe that VideoPlayer would try to decode before my changes. I previously merged this change into my intra prediction commit. I'll split this off so it's clearer why I removed it:

https://github.com/SerenityOS/serenity/commit/4173f3427223aadb9d376e5427bd07e6dc53d962#diff-802bcbd4f9ce71bcc5674a95fea292fc8c4b2a98a8476dc1cfad4eae09073ad7L1424